db/migrate/20170410130134_copy_server_id_from_jobs_to_miq_tasks.rb

```diff
@@ -0,0 +1,21 @@
+class CopyServerIdFromJobsToMiqTasks < ActiveRecord::Migration[5.0]
+  class Job < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class MiqTask < ActiveRecord::Base; end
+
+  def up
+    say_with_time("Copying miq_server_id from jobs table to miq_tasks") do
+      Job.where.not(:miq_task_id => nil).find_each do |job|
+        MiqTask.find(job.miq_task_id).update_attributes!(:miq_server_id => job.miq_server_id)
```

Comment:
This might an expensive N+1 during a migration if there are a lot of jobs x tasks.  `.includes` would solve the N+1 on the `.find` at least.